src/utilities/endpoints/endpoints.ts

```diff
@@ -18,6 +18,13 @@ export const publicEndpoints = {
   'BOTLabs Trusted Entity': 'wss://spiritnet.kilt.io',
 };
 
+const backend = 'https://testnet-did-promo.sporran.org';
+
+export const backendEndpoints = {
+  promoStatus: `${backend}/promo_status`,
+  createDid: `${backend}/create_did`,
+};
+
```

Comment:
We can already prepare that we will have a production backend as well

Reply:
By prepare do you mean add a TODO?

Reply:
We can already configure two backends (for now having the same domain), and switch between them depending for example on production or non-production environment, or depending on the currently selected blockchain endpoint.

Reply:
Yeah, I see that you did that already. The only correction I suggest is to use the testnet address instead of an empty string.